app/components/page-header/component.js

```diff
@@ -14,27 +15,35 @@ function fnOrValue(val, ctx) {
 }
 
 
-export default Ember.Component.extend({
+export default Ember.Component.extend(HoverDropdown, {
   // Inputs
   hasCattleSystem: null,
   currentPath: null,
 
   // Injections
-  projects         : Ember.inject.service(),
-  project          : Ember.computed.alias('projects.current'),
-  k8s              : Ember.inject.service(),
-  namespace        : Ember.computed.alias('k8s.namespace'),
-  projectId        : Ember.computed.alias(`tab-session.${C.TABSESSION.PROJECT}`),
-  namespaceId      : Ember.computed.alias('k8s.namespace.id'),
-  settings         : Ember.inject.service(),
-  access           : Ember.inject.service(),
-  prefs            : Ember.inject.service(),
-  isAdmin          : Ember.computed.alias('access.admin'),
+  projects      : Ember.inject.service(),
+  project       : Ember.computed.alias('projects.current'),
+  k8s           : Ember.inject.service(),
+  namespace     : Ember.computed.alias('k8s.namespace'),
+  projectId     : Ember.computed.alias(`tab-session.${C.TABSESSION.PROJECT}`),
+  namespaceId   : Ember.computed.alias('k8s.namespace.id'),
+  settings      : Ember.inject.service(),
+  access        : Ember.inject.service(),
+  prefs         : Ember.inject.service(),
+  isAdmin       : Ember.computed.alias('access.admin'),
+  hasVm         : Ember.computed.alias('project.virtualMachine'),
+  hasSwarm      : Ember.computed.alias('project.orchestrationState.hasSwarm'),
+  hasKubernetes : Ember.computed.alias('project.orchestrationState.hasKubernetes'),
+  hasMesos      : Ember.computed.alias('project.orchestrationState.hasMesos'),
+  swarmReady    : Ember.computed.alias('project.orchestrationState.swarmReady'),
+  mesosReady    : Ember.computed.alias('project.orchestrationState.mesosReady'),
```

Comment:
It looks like they're just reordered in the file and got doesn't recognize it because  of space auto-formatting... Which is why I don't like  randomly-applied auto-formatting :-)